aunt-leahs-app/src/components/adminLocationList/locationList.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { List, Fab } from '@material-ui/core';
+import { LocationListItem } from './locationListItem.jsx';
+import AddIcon from '@material-ui/icons/Add';
+import CustomButton from '../customButton'
+
+export class LocationList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            locations: [
```

Comment:
This should be moved to constants so we can reuse it in other places in testing

Reply:
I feel like we wouldn't actually take them from the constants files because they really should be from an API call and from there, in testing, we should be mocking the API. But I do see that there could be some use for the time being so I dunno. Thoughts, @codygagnon ?

Reply:
I agreed actually, best not to waste your time designing constants that won't be used! Leaving this in is fine until it gets replaced with a call.

Reply:
As Justin points out, this "constant" will be replaced with data from the API, so it doesn't matter very much where we put it for now. With that said, if it is being used in two or more places at the moment, it would make most sense to put it in the constants file and import it.